apps/chef_index/test/lucene_sexp_tests.erl

```diff
@@ -1,205 +0,0 @@
-    Formats = [{"~saterm", [{op, <<"aterm">>}]},
-               {"first ~ssecond", [<<"first">>, {op, <<"second">>}]},
-               {"~sfirst second", [{op, <<"first">>}, <<"second">>]}],
-    InsertOp = fun(L, Op) when is_list(L) ->
-                       lists:keyreplace(op, 1, L, {Op,?gv(op, L)})
-               end,
-    Tests = [ {?i2b(io_lib:format(In, [Op])), InsertOp(L, ?i2b(Op))}
-              || Op <- Prefixes,
-                 {In, L} <- Formats ],
-    [ ?_assertEqual(E, lucene:parse(I)) || {I, E} <- Tests ].
-
-ignore_inner_special_chars_in_terms_test_() ->
-    Specials = ["+", "-", "*", "?", "_", "."],
-    Terms = [ ?i2b(io_lib:format("foo~sbar", [S])) || S <- Specials ],
-    [ ?_assertEqual([T], lucene:parse(T)) || T <- Terms].
-
-phrase_query_test_() ->
-    Phrases = [{"\"single\"", {phrase, <<"single">>}},
-               {"\"one two three\"", {phrase, <<"one two three">>}},
-               {"\"has \\\"escaped\\\" quotes\\\"s\"",
-                {phrase, <<"has \\\"escaped\\\" quotes\\\"s">>}},
-               % required phrase
-               {"+\"one two\"", {<<"+">>, {phrase, <<"one two">>}}},
-               % prohibited phrase
-               {"-\"one two\"", {<<"-">>, {phrase, <<"one two">>}}}
-               ],
-    [ ?_assertEqual(E, hd(lucene:parse(I))) || {I, E} <- Phrases ].
-
-not_queries_on_phrases_test_() ->
-    Queries = [
-               {"a NOT \"b c\"", [<<"a">>, {'NOT', {phrase, <<"b c">>}}]},
-               {"a !\"b c\"", [<<"a">>, {'!', {phrase, <<"b c">>}}]},
-               {"a ! \"b c\"", [<<"a">>, {'!', {phrase, <<"b c">>}}]}],
-    [ ?_assertEqual(E, lucene:parse(I)) || {I, E} <- Queries ].
-
-field_labeled_queries_test_() ->
-    Tests = [
-             {"a_field:aterm", [[{field, <<"a_field">>}, {term, <<"aterm">>}]]},
-             {"afield:(a b c)",
-              [[{field, <<"afield">>},
-                {term, {group, [<<"a">>, <<"b">>, <<"c">>]}}]]},
-             {"afield:\"a b\"", [[{field, <<"afield">>}, {term, {phrase, <<"a b">>}}]]}
-            ],
-    [ ?_assertEqual(E, lucene:parse(I)) || {I, E} <- Tests ].
-
-term_boosting_test_() ->
-    Tests = [{"word^0.5", {boost, <<"word">>, 0.5}},
-             {"\"one two\"^10", {boost, {phrase, <<"one two">>}, 10}}
-            ],
-    [ ?_assertEqual(E, hd(lucene:parse(I))) || {I, E} <- Tests ].
-
-fuzzy_query_test_() ->
-    Tests = [{"word~", {fuzzy, <<"word">>}},
-             {"word~0.5", {fuzzy, <<"word">>, 0.5}}],
-    [ ?_assertEqual(E, hd(lucene:parse(I))) || {I, E} <- Tests ].
-
-fielded_fuzzy_query_test_() ->
-    Tests = [{"afield:word~", [{field, <<"afield">>}, {term, {fuzzy, <<"word">>}}]},
-             {"afield:word~0.5", [{field, <<"afield">>}, {term, {fuzzy, <<"word">>, 0.5}}]}],
-    [ ?_assertEqual(E, hd(lucene:parse(I))) || {I, E} <- Tests ].
-
-%% TODO: support for proximity queries?
-%%     {"\"one two\"~10", {proximity, {phrase, <<"one two">>}, 10}}],
-%% as well as then a fielded version
-
-% TODO
```

Comment:
While I know these modules being tested are generated, is there value in keeping the tests to run against the generated files?  It's effectively a test of the PEG, so I'm reluctant to discard it entirely.